localization:

  # NDT Topics
  # pointCloudTopic: "/velodyne_points"               # Point cloud data
  pointCloudTopic: "/carla/lidar1/point_cloud"               # Point cloud data
  imuOdomTopic: "odometry/imu"      # IMU pre-preintegration output odometry
  # mapPath: "/home/mrk/Experiment/zetong/total_map.pcd"
  mapPath: "/home/mrk/Experiment/yunle/changed_GlobalMap.pcd"
  # mapPath: "/home/mrk/Experiment/airport/filtered_airport.pcd"

  # imu use or not
  # use_imu: true
  use_imu: true

  # imu preintegration Topics
  # imuTopic: "/imu"                     # imu raw data
  imuTopic: "/carla/imu1/imu"                     # imu raw data
  ndtOdomTopic: "ndt/current_pose_est" # ndt matching output odometry
  
  # Frames
  # lidarFrame: "velodyne"
  lidarFrame: "lidar1"
  baselinkFrame: "base_link"
  odometryFrame: "odom"
  mapFrame: "map"

  # NDT params
  NDTEpsilon: 0.01
  NDTStepSize: 0.1
  NDTMaxIteration: 35
  NDTResolution: 2.0
  NDTCurrentPointsLeafSize: 0.4


  # IMU Settings
  imuAccNoise: 3.9939570888238808e-03
  imuGyrNoise: 1.5636343949698187e-03
  imuAccBiasN: 6.4356659353532566e-05
  imuGyrBiasN: 3.5640318696367613e-05
  imuGravity: 9.80511
  imuRPYWeight: 0.01

  # Extrinsics: T_lb (lidar -> imu)
  extrinsicTrans: [0.0, 0.0, 0.0]
  extrinsicRot: [ 1, 0, 0,
                  0, 1, 0,
                  0, 0, 1]

  # extrinsicRot: [1, 0, 0,
  #                 0, 1, 0,
  #                 0, 0, 1]
  # extrinsicRPY: [1, 0, 0,
  #                 0, 1, 0,
  #                 0, 0, 1]



